index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header class="toolbar_class">
+      <div class="logo_container">
+        <p class="logo_class">My Logo</p>
+      </div>
+      <div class="option_bar">
+        <img src="Union.png" alt="union" class="union-class">
+      </div>
+    </header>
+
+    <div class="background">
+      <div class="intro_class">
+        <p class="identity_text">I'm Simba, I'm glad to see you!</p>
+        <p class="description_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="connect_text">LET'S CONNECT</p>
+        <div class="social_media_icons">
+          <img src="twitter.png" class="icons" alt="twitter">
+          <img src="github.png" class="icons" alt="github">
+          <img src="Linkedin_icon.png" class="icons" alt="linkedin">
```

Comment:
- [ ] Good job so far by implementing the design almost similar to the Figma design. But, to make your code more professional, I think you should implement the list of items by using the `ul` tag to the group and each element a ` li` . Add also a working link with `a` tag for each element.